packages/atomic/src/pages/index.html

```diff
@@ -128,6 +128,10 @@
       atomic-search-layout {
         row-gap: var(--atomic-layout-spacing-y);
       }
+
+      atomic-layout-section[section='popovers'] {
+        display: flex;
+      }
```

Comment:
should be moved to packages/atomic/src/components/search/atomic-layout/atomic-search-layout.pcss

Reply:
But popovers is an arbitrary section name. Shouldn't clients be able to name their section whatever name they'd like?

Reply:
We offer a basic search layout with predefined names & styling. https://docs.coveo.com/en/atomic/latest/usage/layouts/

Customer have the choice of using it or not, but we should provide a section for these components. Could be a section for all the horizontal facets, not just he popovers.

I'd name the section `horizontal-filters` & move all the horizontal facets inside, similar to the mockups:
<img width="1643" alt="Screen Shot 2022-08-04 at 12 20 48 PM" src="https://user-images.githubusercontent.com/4923043/182899790-9a920477-4829-4b74-b94d-327ff0558ec2.png">

Reply:
I see !